pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -63,14 +69,15 @@ func GetTaskFuncFromTaskRun(ctx context.Context, k8s kubernetes.Interface, tekto
 			}, nil
 		}, nil
 	}
-	return GetTaskFunc(ctx, k8s, tekton, taskrun.Spec.TaskRef, taskrun.Namespace, taskrun.Spec.ServiceAccountName)
+	return GetTaskFunc(ctx, k8s, tekton, requester, taskrun, taskrun.Spec.TaskRef, "", taskrun.Namespace, taskrun.Spec.ServiceAccountName)
```

Comment:
what's the reason to use "" here instead of taskrun.name?

Reply:
In practice in this case, it doesn't really matter - if the `targetName` is `""`, then `owner.Name` (in this case, `taskrun.name`) is used. I'm not a huge fan of the signature for `GetTaskFunc`, but can't come up with something cleaner at the moment - I'll try to come back and refactor `GetTaskFunc` in general if and when I get a good idea. =) That said, I'll change this to use `taskrun.Name` for now, even though it's unnecessary.